src/Control/Effect/Sum.hs

```diff
@@ -20,26 +22,59 @@ instance (HFunctor f, HFunctor g) => HFunctor (f :+: g)
 instance (Effect f, Effect g)     => Effect   (f :+: g)
 
 
-class Member (sub :: (* -> *) -> (* -> *)) sup where
-  inj :: sub m a -> sup m a
-  prj :: sup m a -> Maybe (sub m a)
+type Member (sub :: (* -> *) -> (* -> *)) sup = MemberAt (PathTo sub sup) sub sup
```

Comment:
I had originally left defined `Member` as a class, and provided a single instance:

```haskell
instance (PathTo sub sup ~ path, MemberAt path sub sup) => Member sub sup where
  inj = inj' @path
  prj = prj' @path
```

However, this resulted in warnings about brittle inference suggesting enabling `MonoLocalBinds` at each and every reference of `Member`.

Fortunately, the constraint synonym avoids this, and since we don’t need to introduce a new type variable to represent the path, we can further avoid any need for e.g. `QuantifiedConstraints`.